apps/dashboard/app/apps/ood_app_group.rb

```diff
@@ -1,9 +1,10 @@
 class OodAppGroup
   attr_accessor :apps
 
-  def initialize(title: "", apps: [])
+  def initialize(title: "", apps: [], nav_limit: nil)
     @apps = apps
-    @_title = title
+    @title = title
```

Comment:
I guess we should just move the title back to an `attr_accessor` and remove the method.  I changed it around it in 2d09f531fc4c2bdcac78e7e607f40441e2294daa and added the function but if it's not being computed seems like it'd raise questions in the future.